packages/filecoin-actor-utils/src/utils.ts

```diff
@@ -0,0 +1,45 @@
+import { networkActorCodeMap } from "./data"
+
+/**
+ * Resolves the actor name by providing the actor code. When also providing
+ * the network name, it will limit the search to that specific network.
+ * @param actorCode the actor code which is used to resolve the actor name
+ * @param networkName (optional) the network in which to search for the actor name 
+ * @returns the actor name when found or null when not found
+ */
+export const getActorName = (actorCode: string, networkName?: string): string | null => {
+  
+  // If "networkName" is provided, find the
+  // actor name for that specific network
+  if (networkName) {
+    const actorCodeMap = networkActorCodeMap[networkName]
+    for (const [name, code] of Object.entries(actorCodeMap)) {
+      if (code === actorCode) return name
+    }
+  }
```

Comment:
Do we not want to create some type of reflection for the actorCodes like we have before so we don't have to iterate here? Seems like with the FVM, we might have a much longer list of actors to go through.

For now it probably doesn't matter, just thinking out loud

Reply:
Yeah, fair point. It should be such a light lookup that it really won't make a difference at this point. If we manage to store the list of custom actors here that list might get a bit longer. We can reverse the Map after loading in keep it in memory. But when we get to that point, this might need some overhaul altogether.